<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>780</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>780</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollBar{
            border: 0px solid #999999;
            background:rgb(134, 134, 134);
            width:10px;    
            margin: 0px 0px 0px 0px;
        }
        QScrollBar::handle{         
       
            min-height: 0px;
          	border: 0px solid red;
			border-radius: 4px;
			background-color: black;
        }
        QScrollBar::add-line{       
            height: 0px;
            subcontrol-position: bottom;
            subcontrol-origin: margin;
        }
        QScrollBar::sub-line{
            height: 0 px;
            subcontrol-position: top;
            subcontrol-origin: margin;
        }</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">QFrame{
	background:rgb(66,66,66);
	border-radius:25px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,20">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QFrame" name="TitleBarFrame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>1</number>
                </property>
                <property name="topMargin">
                 <number>1</number>
                </property>
                <property name="rightMargin">
                 <number>1</number>
                </property>
                <property name="bottomMargin">
                 <number>1</number>
                </property>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="styleSheet">
                   <string notr="true">QLabel{font-size:15px;
		font-weight:bold;
color:white;}</string>
                  </property>
                  <property name="text">
                   <string>WallpaperChanger_0.1</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="underBtn">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton{background:rgb(93,93,93); border-radius:25px;}
QPushButton::hover{background:rgb(255, 85, 0);border-radius:25px;}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="maximizeBtn">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton{background:rgb(93,93,93); border-radius:25px;}
QPushButton::hover{background:rgb(255, 85, 0);border-radius:25px;}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="exitBtn">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton{background:rgb(93,93,93); border-radius:25px;}
QPushButton::hover{background:rgb(255, 85, 0);border-radius:25px;}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="FrameVLayout" stretch="20,2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2" stretch="10,0,20">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="leftMargin">
                <number>2</number>
               </property>
               <item>
                <widget class="QListWidget" name="wallpaperList">
                 <property name="styleSheet">
                  <string notr="true">QListWidget::item {
    color:rgb(255, 255, 255);
	border:1px solid #dddddd;
}
QListWidget::hover{
	
}</string>
                 </property>
                 <property name="selectionRectVisible">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="imgLabel">
                 <property name="minimumSize">
                  <size>
                   <width>560</width>
                   <height>380</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel{
	border:1px solid rgb(200,200,200);
	border-radius: 0px;
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0">
               <item>
                <widget class="QLineEdit" name="currentWeatherInfoLabel">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLineEdit{border-radius:25px;
		color:rgb(52, 52, 52);
		font-size:30px;
		font-weight:bold;
		font-style:italic;
		background:rgb(198, 176, 163);			
}</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="placeholderText">
                  <string>현재날씨</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTabWidget" name="tabWidget">
                 <property name="styleSheet">
                  <string notr="true">QTabWidget::pane {
    border: 1px solid white;
    background: white;
	
}

QWidget{background-color:rgb(66,66,66);}

QTabWidget::tab-bar:top {
    top: 1px;
}

QTabWidget::tab-bar:bottom {
    bottom: 1px;
}

QTabWidget::tab-bar:left {
    right: 1px;
}

QTabWidget::tab-bar:right {
    left: 1px;
}

QTabBar::tab {
    border: 1px solid black;
}

QTabBar::tab:selected {
    background: white;
}

QTabBar::tab:!selected {
    background: silver;
}

QTabBar::tab:!selected:hover {
    background: #999;
}

QTabBar::tab:top:!selected {
    margin-top: 3px;
}

QTabBar::tab:bottom:!selected {
    margin-bottom: 3px;
}

QTabBar::tab:top, QTabBar::tab:bottom {
    min-width: 8ex;
    margin-right: -1px;
    padding: 5px 10px 5px 10px;
}

QTabBar::tab:top:selected {
    border-bottom-color: none;
}

QTabBar::tab:bottom:selected {
    border-top-color: none;
}

QTabBar::tab:top:last, QTabBar::tab:bottom:last,
QTabBar::tab:top:only-one, QTabBar::tab:bottom:only-one {
    margin-right: 0;
}

QTabBar::tab:left:!selected {
    margin-right: 3px;
}

QTabBar::tab:right:!selected {
    margin-left: 3px;
}

QTabBar::tab:left, QTabBar::tab:right {
    min-height: 8ex;
    margin-bottom: -1px;
    padding: 10px 5px 10px 5px;
}

QTabBar::tab:left:selected {
    border-left-color: none;
}

QTabBar::tab:right:selected {
    border-right-color: none;
}

QTabBar::tab:left:last, QTabBar::tab:right:last,
QTabBar::tab:left:only-one, QTabBar::tab:right:only-one {
    margin-bottom: 0;
}</string>
                 </property>
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="tab">
                  <attribute name="title">
                   <string>날씨</string>
                  </attribute>
                  <layout class="QVBoxLayout" name="verticalLayout_8">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
                     <property name="leftMargin">
                      <number>1</number>
                     </property>
                     <property name="topMargin">
                      <number>1</number>
                     </property>
                     <property name="rightMargin">
                      <number>1</number>
                     </property>
                     <property name="bottomMargin">
                      <number>1</number>
                     </property>
                     <item>
                      <widget class="QWebEngineView" name="webEngineView"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="tab_2">
                  <attribute name="title">
                   <string>날씨 추이</string>
                  </attribute>
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QTableWidget" name="detailedConditionTable">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QTabWidget{
	background: white;
}</string>
                     </property>
                     <property name="sizeAdjustPolicy">
                      <enum>QAbstractScrollArea::AdjustIgnored</enum>
                     </property>
                     <property name="rowCount">
                      <number>1</number>
                     </property>
                     <property name="columnCount">
                      <number>5</number>
                     </property>
                     <attribute name="horizontalHeaderCascadingSectionResizes">
                      <bool>false</bool>
                     </attribute>
                     <row/>
                     <column/>
                     <column/>
                     <column/>
                     <column/>
                     <column/>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QCheckBox" name="backgroundCheckBox">
               <property name="text">
                <string>백그라운드에서 실행</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>70</height>
                </size>
               </property>
               <property name="title">
                <string>날씨 계산 시간(분)</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <property name="leftMargin">
                   <number>5</number>
                  </property>
                  <property name="rightMargin">
                   <number>5</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="intervalSliderValue">
                    <property name="minimumSize">
                     <size>
                      <width>30</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>1</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSlider" name="intervalSlider">
                    <property name="minimumSize">
                     <size>
                      <width>250</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>60</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="intervalBtn">
                    <property name="minimumSize">
                     <size>
                      <width>50</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton{background:rgb(93,93,93); border-radius:25px;}
QPushButton::hover{background:rgb(10, 251, 255); border-radius:5px;}
</string>
                    </property>
                    <property name="text">
                     <string>확인</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
